test/WebJobs.Script.Tests/WebScriptHostManagerTests.cs

```diff
@@ -283,7 +283,7 @@ public Fixture()
                     "Info Host lock lease acquired by instance ID '(.+)'",
                     "Info Function 'Excluded' is marked as excluded",
                     @"Info Generating ([0-9]+) job function\(s\)",
-                    @"Info Starting Host \(HostId=function-tests-node, Version=(.+), ProcessId=[0-9]+, Debug=False\)",
+                    @"Info Starting Host \(HostId=function-tests-node, Version=(.+), ProcessId=[0-9]+, Debug=False\) (.+)",
```

Comment:
I thing @mathewc is referring to the regex change here, right? Are you adding the version to the end? It's already in the output.